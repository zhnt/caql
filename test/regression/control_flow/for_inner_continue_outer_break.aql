// Nested loops: inner continue, outer break
let sum = 0
for i = 1, 5, 1 {
    if i == 3 { break }
    for j = 1, 5, 1 {
        if j == 3 { continue }
        sum = sum + i * 10 + j
        print(i * 10 + j)
    }
}
print(sum)
