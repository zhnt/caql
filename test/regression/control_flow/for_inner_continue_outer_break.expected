11
12
14
21
22
24
104
